src/main/java/helpers/Randomize.java

```diff
@@ -0,0 +1,19 @@
+package helpers;
+
+import java.security.SecureRandom;
+
+public final class Randomize {
+    private static final String SEQUENCE = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+    private static SecureRandom rnd = new SecureRandom();
+
+    public static String randomString(int length){
```

Comment:
Take a look on apache-commons lib for string. It will have StringUtils class with pre-defined randomization.